comiccollector/templates/topnav.html

```diff
@@ -1,39 +1,41 @@
-<div ng-cloak>
-  <div class="fixed">
-    <top-bar>
-      <ul class="title-area">
-        <li class="name">
-          <h1><a href="{% url 'home' %}">Comic Collector</a></h1>
-        </li>
-        <li toggle-top-bar class="menu-icon"><a href="#">Menu</a></li>
-      </ul>
+<nav
+    class="top-bar"
+    data-topbar
+    role="navigation"
+    ng-cloak>
```

Comment:
Actually it not needed, but it prevents angular template tags from showing before they've been rendered anywhere in that div